Integrations/DomainTools_Iris/DomainTools_Iris.py

```diff
@@ -0,0 +1,646 @@
+    domain_registrant = domain_result.get('registrar').get('value') if isinstance(domain_result.get('registrar'),
+                                                                                  dict) else domain_result.get(
+        'registrar')
+    registrar_status = domain_result.get('registrar_status')
+    ip_data = domain_result.get('ip')
+    ip_country_code = domain_result.get('ip')[0] if len(domain_result.get('ip')) else ''
+    mx_servers = domain_result.get('mx')
+    spf_info = domain_result.get('spf_info')
+    ssl_certificates = domain_result.get('ssl_info')
+    redirects_to = domain_result.get('redirect')
+
+    domain_tools_context = {
+        'Name': domain,
+        'LastEnriched': datetime.now().strftime('%Y-%m-%d'),
+        'Analytics': {
+            'OverallRiskScore': overall_risk_score,
+            'ProximityRiskScore': proximity_risk_score,
+            'ThreatProfileRiskScore': {'RiskScore': threat_profile_risk_score,
+                                       'Threats': threat_profile_threats,
+                                       'Evidence': threat_profile_evidence},
+            'WebsiteResponseCode': website_response,
+            'AlexaRank': alexa_rank,
+            'Tags': tags
+        },
+        'Identity': {
+            'RegistrantName': registrant_name,
+            'RegistrantOrg': registrant_org,
+            'RegistrantContact': registrant_contact,
+            'SOAEmail': soa_email,
+            'SSLCertificateEmail': ssl_email,
+            'AdminContact': admin_contact,
+            'TechnicalContact': technical_contact,
+            'BillingContact': billing_contact,
+            'EmailDomains': email_domains,
+            'AdditionalWhoisEmails': additional_whois_emails
+        },
+        'Registration': {
+            'DomainRegistrant': domain_registrant,
+            'RegistrarStatus': registrar_status,
+            'DomainStatus': domain_status,
+            'CreateDate': create_date,
+            'ExpirationDate': expiration_date
+        },
+        'Hosting': {
+            'IPAddresses': ip_data,
+            'IPCountryCode': ip_country_code,
+            'MailServers': mx_servers,
+            'SPFRecord': spf_info,
+            'NameServers': name_servers,
+            'SSLCertificate': ssl_certificates,
+            'RedirectsTo': redirects_to,
+            'GoogleAdsenseTrackingCode': google_adsense,
+            'GoogleAnalyticTrackingCode': google_analytics
+        }
+    }
+
+    domain_context = {
+        'Name': domain,
+        'DNS': ip_addresses,
+        'Vendor': 'DomainTools',
+        'CreationDate': create_date,
+        'RiskScore': overall_risk_score,
+        'DomainStatus': domain_status,
+        'ExpirationDate': expiration_date,
+        'NameServers': domain_name_servers,
+        'Registrant': domain_registrant_contact
+    }
+    if overall_risk_score and overall_risk_score >= RISK_THRESHOLD:
```

Comment:
it is better to check if the DBot Score is 3, and then init the `Malicious` object